File: Modul 5/SOAL 5/Bahasa Python/PRAK405-2210817210028-Muhammad Aulia Rasyid.py
a, b = input().split()
a = int(a) ; b = int(b)
for i in range(1, a + 1) :
    j = i 
    while j > 1 :
        print("(%d * %d) + "%(j, b), end='') ; j -= 1
    x = 1 ; hasil = (i * b)
    while x < i :
        hasil += (x * b) ; x += 1
    print("(%d * %d) = %d"%(j, b, hasil))
y = 1 ; z = 1 ; total = 0
while y <= a :
    total += z * b ; z += y + 1 ; y += 1
print("%d"%(total))

File: Modul 5/SOAL 5/Bahasa C/PRAK405-2210817210028-Muhammad Aulia Rasyid.c
#include <stdio.h>
int main ()
{
    int a, b, i, j, x, y, z, hasil, total ;
    scanf("%d %d", &a, &b);
    for (i = 1; i <= a; i++){
        for (j = i; j > 1; j--){
            printf("(%d * %d) + ", j, b); }
        for (x = 1, hasil = x * b; x < i; x++, hasil += (x * b)){}
        printf("(%d * %d) = %d\n", j, b, hasil); }
    for (y = 1, z = 1, total = 0; y <= a; z += y + 1,  y++) {
        total += z * b ; }
    printf("%d", total);
}
